classes/helper.php

```diff
@@ -815,8 +789,10 @@ public static function handle_replace_csv(string $data, string $type = 'db') {
                 $rowskip++;
             } else if ($type == 'db') {
                 // Replace the string.
-                self::replace_text_in_a_record($record[$tableindex], $record[$columnindex],
-                    $record[$matchindex], $record[$replaceindex], $record[$idindex]);
+                if (!self::replace_text_in_a_record($record[$tableindex], $record[$columnindex],
+                    $record[$matchindex], $record[$replaceindex], $record[$idindex])) {
+                        $rowskip++;
```

Comment:
I don't think these are skipped, skipped means we didn't supply a replacement. These had a replacement which didn't work, we need a whole extra var to store this count